app/default/data.py

```diff
@@ -194,7 +205,11 @@ def get_fund_data_by_short_name(fund_short_name, as_dict=False):
         return Fund.from_dict(fund_response)
 
 
-def get_round_data(fund_id, round_id, language=None, as_dict=False):
+@lru_cache(maxsize=5)
+def get_round_data(
+    fund_id, round_id, language=None, as_dict=False, ttl_hash=None
```

Comment:
For consistency, these should probably be separated onto their own lines, as [seen below](https://github.com/communitiesuk/funding-service-design-frontend/pull/366/files#diff-25ffeea551d125947892e07d498380fe030e5f7c005ba15e165622de5d567f0eR242)

Reply:
we don't explicitly separate the lines for small number of argument in a function. --max-line-length=120 is set in hooks, the consistency is handled by the pre-commit hooks